Resources/Locale/en-US/station-goal/station-goals.ftl

```diff
@@ -0,0 +1,224 @@
+    2. When captured fauna dies, you are required to catch another, cloning is strictly prohibited.
+
+    Once you collect the required fauna, you must study them and write a report on their properties.
+    The report must be stamped by the head of the department and faxed to Central Command.
+
+    Experience Requirements:
+    1. Experience should be documented in detail;
+    2. Test activities may include: working with gases, smoke, foam, or injecting experimental reagents (e.g. Cognizine) into captured fauna.
+
+station-goal-museum=
+    Dear Command of the station, the purpose of your shift is to build a museum, the exhibits for which will be unique objects collected from the station.
+
+    Below are the requirements for the design of the museum:
+    1. The museum must be structurally connected to the station by a space-protected corridor, or be located within it;
+    2. The premises must be of a size that allows them to easily receive a large number of visitors;
+    3. The premises must be provided with a standard atmosphere, ventilation and stable power supply;
+    4. Room decoration should be visually pleasing;
+    5. Exhibits must be reinforced accordingly to what is contained in them.
+
+    Exhibit requirements:
+    1. Exhibits must be unique in their kind;
+    2. Each department must provide at least 2 exhibits for the museum fund;
+    3. The total number of exhibits must be at least 20.
+
+    Exhibits may include:
+    1. Exotic drinks and dishes that require an extraordinary method of production and/or non-standard ingredients;
+    2. Exotic matter/substance;
+    3. Works of art (e.g. statues, paintings);
+    4. Fully studied and documented artifacts (optionally provide a copy of the document);
+    5. High-tech devices or tools;
+    6. High-tech or high-power weapons;
+    7. Robotic entities (e.g. Mechs, Cyborgs, Drones);
+    8. Mutated biological organisms;
+    9. Domesticated wild animals or intelligent non-humanoid life forms;
+    10. Found treasures or items not available on the market.
+
+    Upon completion of the museum, it is required to provide the crew with at least 20 minutes of free time from work so that they can visit the museum.
+
+station-goal-area=
+    Dear Station Command, the goal of your shift is to increase the effective use of space at the station.
+
+    It is required to bring the abandoned premises into proper form and find a use for them.
+    Each department must equip and effectively use the area of adjacent maintenance tunnels.
+    Sufficiently spacious maintenance tunnels need to be converted into residential areas.
+    The remaining tunnels should be provided with floor coverings and adequate lighting.
+    In addition, it is necessary to provide a public, well-lit corridor connecting all the restored compartments and new bedrooms.
+
+station-goal-bureaucratic-error =
+    ACCESS TO THIS DOCUMENT IS PROHIBITED FOR PERSONS WHO DO NOT HAVE LEGAL IMMUNITY
```

Comment:
Not entirely sure what this means; given no one has legal immunity under any sort of Space Law (Delta or Wiz)